docs/product/traces/index.mdx

```diff
@@ -0,0 +1,94 @@
+
+Adding one set of span conditions let's you search all spans.
+
+The following example shows you all traces where:
+- A user with the email `foo@example.com` loaded a page on your frontend on a specific route
+
+![View of trace explorer with one span conditions set](./img/traces-one-span-conditions-example.png)
+
+Alternatively, adding multiple span conditions will only return traces that contain ALL span conditions.
+
+The following example let's you find a trace where:
+- A user with the email `foo@example.com` loaded a page on your frontend on a specific route
+- AND within that same trace, there was a slow database span
+
+![View of trace explorer with multple span conditions set](./img/traces-multiple-span-conditions-example.png)
+
+### Filter by Environment, and Date Range
+
+These common filters allow you to filter on your environments, as well as specify the date range you want to zoom in on.
+
+Since traces are distributed in nature, there is no page-wide project filter, but you can filter on projects per set of span conditions by adding `project:your-project-here` in the search.
+
+### Filter by Search Conditions
+
+<Note>
+
+When performing searches on spans, based on your search criteria
+and sample rate, the events available may be limited because Traces uses
+sampled data only. Learn more by reading about [our retention priorities](/product/performance/retention-priorities/).
+
+</Note>
+
+All events have built-in key fields or custom tags, this includes our span data. Use the search bar to enter these keys and assign them values. This will filter down your list of events. For a list of the built-in key fields, check out our [Searchable Properties for spans documentation](/product/sentry-basics/search/searchable-properties/spans).
+
+### Syntax
+
+The Query Builder syntax is identical to [Sentry's Search syntax](/product/sentry-basics/search/). After you enter a key field from above or a custom tag, you can use any of the referenced syntax. For example, `count()` gives you the number of times an event occurs. This can be written in the following ways:
+
+- Exact match (is equal to): `span.duration:99`
+- Upper bounds (is less than or equal to): `span.duration:<99` or `span.duration:<=99`
+- Lower bounds (is more than or equal to): `span.duration:>99` or `span.duration:>=99`
+- Multiple bounds (is more and less than): `span.duration:>10 span.duration:<20`
+
+Use `OR` and `AND` search conditions between filters. However `OR` cannot be used between aggregate and non-aggregate filters. For more details about these conditions, see [Using `OR` and `AND`](/product/sentry-basics/search/#using-or-and-and).
+
+You can also search multiple values for the same key by putting the values in a list. For example, "x:[value1, value2]" will find the same results as "x:value1 `OR` x:value2". You can learn more in our content covering [Multiple Values on the Same Key](/product/sentry-basics/search/#multiple-values-on-the-same-key).
+
```

Comment:
```suggestion
## Syntax

The Traces Query Builder syntax is identical to [Sentry's Search syntax](/product/sentry-basics/search/). After you enter a key field from above or a custom tag, you can use any of the referenced syntax. For example, `count()` gives you the number of times an event occurs. This can be written in the following ways:

- Exact match (is equal to): `span.duration:99`
- Upper bounds (is less than or equal to): `span.duration:<99` or `span.duration:<=99`
- Lower bounds (is more than or equal to): `span.duration:>99` or `span.duration:>=99`
- Multiple bounds (is more and less than): `span.duration:>10 span.duration:<20`

You can use `OR` and `AND` search conditions between filters. However, `OR` cannot be used between aggregate and non-aggregate filters. For more details about these conditions, see [Using `OR` and `AND`](/product/sentry-basics/search/#using-or-and-and).

You can also search multiple values for the same key by putting the values in a list. For example, "x:[value1, value2]" will find the same results as "x:value1 `OR` x:value2". You can learn more by reading about [Multiple Values on the Same Key](/product/sentry-basics/search/#multiple-values-on-the-same-key).

```